tests/frame.test.js

```diff
@@ -0,0 +1,236 @@
+const Frame = require("../main/frame");
+
+describe("frame", () => {
+  it("has a score property of 0", () => {
```

Comment:
Jest has a beforeEach function that helps minimize setup repetitions, like instantiating the class you're testing